expressEngine/product/service/product.js

```diff
@@ -25,6 +25,27 @@ class Service {
     }
     return { data: product, message: success.productRegistered };
   }
+
+  getProduct = async ({ query, body }) => {
+    const dataToFind = {
+      collection: Product,
+      data: {
+        ...body,
+        isDiscontinued: undefined,
```

Comment:
@aman-successive Either don't set it or set it as false/true depending on the condition

Reply:
changed it